### ISSUE TITLE
[turborepo] deployment problems when using remote caching

### ISSUE THREAD

**Issue by wimluk** [vercel/turbo#4903]
### What version of Turborepo are you using?

1.9.3

### What package manager are you using / does the bug impact?

npm

### What operating system are you using?

Windows

### Describe the Bug

After the initial setup (npx create-turbo@latest), enabling remote caching (npx turbo login and npx turbo link) and building locally I am able to deploy the starter app docs (apps/docs) to vercel with a cache hit and no other problems. After adding a simple div with some text in apps/docs/app/page.tsx and building locally I get the following error when deploying:

13:18:46.623 | Error: ENOENT: no such file or directory, lstat '/vercel/path0/apps\docs/.next\routes-manifest.json'
13:18:47.252 | Deployment completed
13:18:46.801 | ENOENT: ENOENT: no such file or directory, lstat '/vercel/path0/apps\docs/.next\routes-manifest.json'

### Expected Behavior

Successful redeployment with a cache hit like the initial deployment.

### To Reproduce

- initial setup npx create-turbo@latest
- enabling remote caching (npx turbo login and npx turbo link)
- building locally so we get a cache hit
- deploy the starter app docs (apps/docs) to vercel without any changes of the configurations
- get a cache hit and successful deployment without any problems
- add a div to the docs app page
- building locally so we get a cache hit
- publish again
- get an error during deployment

**Comment by nathanhammond** [comment]
Hey @wimluk! We really appreciate you filing this issue. We're addressing it with highest priority.

1.9.5-canary.0 has been released with a change that may have fixed it, but it was authored before your report. #4634

I've also created #4919 which is really the "other half" of 4634.

On my Monday (HKT) I'm going to be writing lots of tests to constrain correctness.

1.9.5, when it gets released, will contain the fix.
